lib/processors/stringEscaper.js

```diff
@@ -0,0 +1,59 @@
+const escapeUnicode = require("escape-unicode");
+
+/**
+ * @see https://ascii.cl/
+ * @type {number}
+ */
+const NUMBER_OF_ASCII_CHARACTERS = 127;
```

Comment:
128
rename to something like CHAR_CODE_OF_LAST_ASCII_CHARACTER ?

Reply:
done